test/integration/lib/philipsHue_server.js

```diff
@@ -0,0 +1,83 @@
+var express = require ('express');
+var bodyParser = require('body-parser');
+var path = require('path');
+
+var philipsHue_server = (function() {
+
+	var welcome_page = '/html/philips_initial.html';		// Philips Hub webpage
+	var suc_resp = require('./success_response.js');		// TBD: move this to appropriate module
+	var api_resp = require('./json/bridge_status.json');	// full status response
+	var light_status_resp = require('./json/lights_status.json'); // light status response
+	
+	var commanded_light_state ={1:true,2:true,3:true};		// TBD: move this to appropriate module
+	var last_received_cmd = null;		
+
+	return {
+
+		setlight: function(id,value) {  // Set the on/off status of simulated light
```

Comment:
I think we can make the meaning of `value` clearer, with something to what we did with BinaryButtons in Gaiatest. What do you think of `turnOnLight(id)`, `turnOffLight(id)` which both use this internal function called `_setLight(id, value)`?